Kernel/ACPI/ACPIParser.cpp

```diff
@@ -58,10 +58,10 @@ ACPIParser::ACPIParser(bool usable)
     s_acpi_parser = this;
 }
 
-ACPI_RAW::SDTHeader* ACPIParser::find_table(const char*)
+PhysicalAddress ACPIParser::find_table(const char*)
 {
     kprintf("ACPI: Requested to search for a table, Abort!\n");
-    return nullptr;
+    return PhysicalAddress((uintptr_t) nullptr);
```

Comment:
Is calling `PhysicalAddress()` constructor will create a `PhysicalAddress` that is equivalent to `PhysicalAddress((uintptr_t) nullptr)`?

Reply:
thanks to @Quaker762 that explained to me in the IRC channel that I can use the default constructor, I'll fix it :)